.github/actions/close-stale.yml

```diff
@@ -0,0 +1,20 @@
+name: "Close stale issues"
+on:
+  schedule:
+    - cron: "0 2 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 65535 # We don't want to mark issues as stale in this action
+          days-before-close: 14
+          stale-issue-label: stale
+          any-of-labels: "needs info"
+          stale-issue-message: |
+            This issue has been automatically marked as stale because it has not had recent activity.
+            It will be closed if no further activity occurs. Thank you for your contributions!
```

Comment:
Guess we can get rid of the state issue message? It should not mark things as stale anyways. Cleans the workflow a bit.

Reply:
Hmm, does it still close issues then... but worth a try!

Reply:
If not, at least shorten it to the bare minimum.